src/libstore/build/local-derivation-goal.cc

```diff
@@ -802,10 +802,8 @@ void LocalDerivationGoal::startBuilder()
 
         userNamespaceSync.create();
 
-        Path maxUserNamespaces = "/proc/sys/user/max_user_namespaces";
-        static bool userNamespacesEnabled =
-            pathExists(maxUserNamespaces)
-            && trim(readFile(maxUserNamespaces)) != "0";
```

Comment:
If the file exists, then we should check that its content is non zero as it would otherwise mean that the namespaces are enabled in the kernel but disabled for the current user